examples/01_molecule_thermo.py

```diff
@@ -2,8 +2,8 @@
 from cantherm.chemistry.molecule import CMol
 
 # Boiler plate to point Cantherm toward it's own data directory
-geom_opt_file = get_sample_file_path("BF3_geom_opt.log")
```

Comment:
Maybe it would make more sense to an another example for this? Something like `02_transition_states.py` where we go over working with the `vibfreqs` and `imag_vibfreqs` attributes of the molecule class.